api/src/schemas/donate/charge-stripe.ts

```diff
@@ -0,0 +1,18 @@
+import { Type } from '@fastify/type-provider-typebox';
+
+export const chargeStripe = {
+  body: Type.Object({
+    amount: Type.Number(),
+    duration: Type.Literal('month'),
+    email: Type.String(),
+    subscriptionId: Type.String()
+  }),
+  response: {
+    200: Type.Object({
+      isDonating: Type.Boolean()
+    }),
+    500: Type.Object({
```

Comment:
```suggestion
    default: Type.Object({
```
This makes sure there's something to serialize the response.